### PR TITLE
Koin Multiplatform with Kotlin 1.4.0

### PR DESCRIPTION
**Pull request by GZaccaroni**
Fixes issue https://github.com/InsertKoinIO/koin/issues/874

### CODE REVIEW

File: koin-projects/gradle.properties [hunk 1]
```diff
@@ -28,4 +28,9 @@ org.gradle.caching=true
 # Bintray
 bintray_repo=koin
 bintray_org=ekito
-bintray_github=https://github.com/touchlab/koin
\ No newline at end of file
+bintray_github=https://github.com/touchlab/koin
```
Comment (arnaudgiuliani) [inline koin-projects/gradle.properties:31]: wrong one, please change it to `https://github.com/InsertKoinIO/koin`

Comment (GZaccaroni) [reply koin-projects/gradle.properties]: Done!

**Comment by nachtien** [comment]
What about addressing this PR in this branch?  https://github.com/InsertKoinIO/koin/pull/873

**Comment by arnaudgiuliani** [comment]
not on the same branch ... I will take time to check that

**Comment by nachtien** [comment]
For measure.kt

```kotlin
/**
 * Measure functions
 *
 * @author Arnaud Giuliani
 */

/**
 * Measure code execution
 */

@OptIn(ExperimentalTime::class)
fun measureDuration(code: () -> Unit): Double {
    return measureTime {
        code()
    }.inMilliseconds
}

fun measureDuration(message: String, code: () -> Unit) {
    val time = measureDuration(code)
    println("$message - $time ms")
}

/**
 * Measure code execution and get result
 */
@OptIn(ExperimentalTime::class)
fun <T> measureDurationForResult(code: () -> T): Pair<T, Double> {
    val result = measureTimedValue(code)

    return Pair(result.value, result.duration.inMilliseconds)
}

fun <T> measureDurationForResult(message: String, code: () -> T): T {
    val (result, time) = measureDurationForResult(code)
    println("$message - $time ms")
    return result
}

private fun Double.inMilliseconds(): Double = (this / 1000000)
```

I _think_ this should work.  But you guys didn't want to use the kotlin libraries for time though so 🤷

**Comment by arnaudgiuliani** [comment]
> I think this should work. But you guys didn't want to use the kotlin libraries for time though so 🤷

It's more that it's breaking each time :/

**Comment by arnaudgiuliani** [comment]
sorry, let me check that @nachtien 👍

**Review by arnaudgiuliani — APPROVED** [review]

**Comment by arnaudgiuliani** [comment]
@nachtien added your snippet to 3.0 branch. Will check to add it to 2.x branch.

thanks @GZaccaroni

**Pull request merged**